components/wb-bubble/_base.scss

```
@@ -0,0 +1,103 @@
+
+/*
+ * Web Experience Toolkit (WET) / Boîte à outils de l'expérience Web (BOEW)
+ * wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * Bubble - Stable CSS
+ */
+/* Animations */
+
```

Include animations below as part of the `.wb-bubble-wrap`.
```suggestion

.wb-bubble-wrap {
```